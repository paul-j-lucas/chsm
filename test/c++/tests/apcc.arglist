-c | char const *a[] | 0
