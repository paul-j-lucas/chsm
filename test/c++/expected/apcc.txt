decl="char const *$[]" name="a"
